packages/vite/src/node/server/hmr.ts

```diff
@@ -213,6 +213,7 @@ export const normalizeHotChannel = (
           name: error.name,
           message: error.message,
           stack: error.stack,
+          ...error, // preserve enumerable properties such as RollupError.loc, frame, plugin
```

Comment:
Alternatively we can return `Error` instance as is `{ e: error }` and let each transport implementation deal with it. This works for default ssr runner, but for others using `JSON.stringify`, it'll end up empty because `JSON.stringify(new Error("hmm")) === '{}'`, so it's certainly a minor inconvenience for them.